cdap-explore-jdbc/src/main/java/co/cask/cdap/explore/jdbc/ExploreStatement.java

```diff
@@ -43,7 +43,7 @@
 public class ExploreStatement implements Statement {
   private static final Logger LOG = LoggerFactory.getLogger(ExploreStatement.class);
 
-  private int fetchSize = 50;
+  private int fetchSize = 1000;
```

Comment:
The memory required is based on the size of an individual row times fetch size. So 1000 might be too big. Based on the row size, we should let the user increase it if required. What do you say?